organisms/organisation-unit-tree/new-features/select-relative-units.feature

```diff
@@ -0,0 +1,18 @@
+Feature: Select units relative to the user's org unit, 'relative units'
+
+  Background:
+    Given the org unit tree has relative selection enabled
+
+  Scenario: Relative selection(s) is made
+    Given the user has opened the relative selection menu
+    When the user selects one or more relative units
```

Comment:
How can the user select more than one unit? As I understand it, he can't select anything, he can choose the depth of the relative selection only?

I'd expect something like this: `When the user selects relative selection depth`

Reply:
It is unintuitive, but it is possible to make multiple relative selections. I don't know the use case, this was implemented _way back when._ It has always been possible to do this, and when I inquired about it I was told it was necessary to keep. See this example, from data-visualizer:
![image](https://user-images.githubusercontent.com/33054985/62112535-2f105380-b2b3-11e9-9d64-89983cedd523.png)
The relative selections are 'user org unit' and 'user sub units', which results in a chart for Sierra Leone(user org unit) and Bo, Bombali...(user sub units). I don't know why this is useful, but apparently it is, heh.

Reply:
I see, I misinterpreted the term "relative units", I thought it referred to org units, my bad